1_demo_hello.bal

```diff
@@ -6,10 +6,21 @@
 
 import ballerina/http;
 
+// Services, endpoint, resources are built into the language
+// This one is HTTP (other options include WebSockets, Protobuf, gRPC, etc)
+// We bind it to port 9090
 service<http:Service> hello bind {port:9090} {
+
+  // The service exposes one resource (hi)
+  // It gets the endpoint that called it - so we can pass response back
+  // and the request struct to extract payload, etc.
```

Comment:
Should we change this to say "`request` object" or just `request` since we no longer use `struct`?